### PR TITLE
Add capability report loop support

### PR DESCRIPTION
**Pull request by leonwanghui**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
design-specs [#1](https://github.com/opensds/design-specs/pull/1)
**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: contrib/drivers/utils/config/config_test.go [hunk 1]
```diff
@@ -30,14 +30,16 @@ func TestParse(t *testing.T) {
...
-				Dedupe:                false,
+				DiskType:        "SSD",
+				AZ:              "ceph",
+				AccessProtocol:  "rbd",
+				ThinProvisioned: true,
+				Compressed:      true,
+				Advanced: map[string]interface{}{
+					"recoveryTimeObjective": 0,
+					"maxIOPS":               1000,
+					"deduped":               false,
```
[the remark this answers is not in the document]
Comment (xing-yang) [reply contrib/drivers/utils/config/config_test.go]: Got it.  Thanks.

[the remark this answers is not in the document]
Comment (leonwanghui) [reply contrib/drivers/utils/config/config_test.go]: Ok, if there is no other question, I will merge it later.

[the remark this answers is not in the document]
Comment (xing-yang) [reply contrib/drivers/utils/config/config_test.go]: It's better for me to merge your patch:).

**Pull request merged**